pkg/envoy/cds/builder.go

```diff
@@ -28,9 +31,139 @@ import (
 	"github.com/openservicemesh/osm/pkg/trafficpolicy"
 )
 
+type clusterBuilder struct {
+	proxyIdentity identity.ServiceIdentity
+
+	outboundMeshTrafficPolicy *trafficpolicy.OutboundMeshTrafficPolicy
```

Comment:
I noticed for all of the *trafficpolicy.*TrafficPolicy objects, we only use the `clusterConfigs`. Can we reduce the stored property to just those fields that are used?